lib/build-secrets/Dockerfile

```diff
@@ -1,8 +1,8 @@
 #TODO: We'll want this to be packaged into a single image,
 #rather than having to install jq every time for example
-FROM balenalib/%%ARCH%%-alpine:3.9
+FROM %%ARCH%%/alpine:3.10
```

Comment:
During the CLI check-in meeting, Cameron asked me why I changed this from `balenalib` to plain Alpine. The original reason was that I thought that some architectures like `balenalib/amd64-alpine` didn't exist, but I was simply doing something wrong as they do exist. Now, having looked at the it a bit further, a new reason to prefer plain Alpine is the image size: both the base image and final build image (after installing bash and jq) are significantly smaller using plain Alpine:

The base image is less than 6MB for `amd64-alpine:3.10`, and about 35MB for `balenalib/amd64-alpine:3.10`. But the latter already includes `bash` whereas the former doesn't, so I checked the final built size too after installing `bash` and `jq`:
```
$ docker images
REPOSITORY                  TAG         IMAGE ID            CREATED              SIZE
amd64-alpine-bash-310       latest      1fe2fea41ff1        4 seconds ago        12.2MB
blib-amd64-alpine-310       latest      ddc2cb0bb164        About a minute ago   38MB
```
The plain Alpine image at ~12MB is about a third the size of the balenalib alternative at ~38MB.

Reply:
This is more than a good enough reason for me :+1: nice work